drivers/storage/portworx/util/util.go

```diff
@@ -508,7 +482,15 @@ func SetupContextWithToken(ctx context.Context, cluster *corev1alpha1.StorageClu
 	}
 
 	// Generate token and add to metadata
-	token, err := GetOperatorToken(cluster, string(pxAuthSecret))
+	claims := &auth.Claims{
+		Issuer:  *cluster.Spec.Security.Auth.Authenticators.SelfSigned.Issuer,
+		Subject: "operator@portworx.io",
+		Name:    "operator communications",
+		Email:   "operator@portworx.io",
+		Roles:   []string{"system.admin"},
+		Groups:  []string{"*"},
+	}
+	token, err := GenerateToken(cluster, pxAuthSecret, claims)
```

Comment:
Should we be using the admin token from the secret instead of creating a new one?

Reply:
Yeah, for now we can use that. Once we bring in the apps issuer change though, we'll probably store the token in memory per app.